config.ts

```diff
@@ -52,10 +55,14 @@ export const config = {
     prefix: `/${pkg.api}`
   },
 
-  redis: {
-    host: process.env.REDIS_EVENTS_PORT_6379_TCP_ADDR || 'localhost',
-    port: +(process.env.REDIS_EVENTS_PORT_6379_TCP_PORT || '6379')
+  redisPurchases: {
+    host: process.env.REDIS_PURCHASES_PORT_6379_TCP_ADDR || 'localhost',
+    port: +(process.env.REDIS_PURCHASES_PORT_6379_TCP_PORT || '6379'),
+    ttl: +(process.env.REDIS_PURCHASES_TTL || 3)
```

Comment:
Redis TTL are expressed in seconds, so I assumed this was seconds.

I discovered while reading the code this are not seconds but days. Either add the units here (like `ttlInDays`, or just use seconds. (and change the place where this is used)